src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -0,0 +1,190 @@
+import { useEffect, useState } from "react";
+import { useLambdasApi } from "../../hooks/use-api";
+import { getHoursAndMinutes } from "../../utils/time-utils";
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import strings from "../../localization/strings";
+
+/**
+ * Component properties
+ */
+interface Props {
+  project : Projects,
+  loggedInpersonId?: number,
+  filter?: string
+}
+
+/**
+ * Task component
+ */
+const TaskTable = ({project, loggedInpersonId, filter}: Props) => {
+  const { tasksApi, timeEntriesApi } = useLambdasApi();
+  const [tasks, setTasks] = useState<Tasks[]>([]);
+  const [timeEntries, setTimeEntries] = useState<number[]>([]);
+  const [open, setOpen] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(false);
+
+  /**
+   * Gather tasks and time entries when project is available and update changes appeared
+   */
+  useEffect(() => {
+    const fetchData = async () => {
+      setLoading(true);
+      if (project && open) {
+        await getTasksAndTimeEntries();
+        setLoading(false);
+      }
+    };
+    fetchData();
+  }, [project, loggedInpersonId, filter, open]);
+
+  /**
+   * Get tasks and total time entries  
+   */
+  const getTasksAndTimeEntries = async () => {
+    try {
+      let fetchedTasks = await tasksApi.listProjectTasks({projectId: project.id});
+      if (loggedInpersonId) {
+        fetchedTasks = fetchedTasks.filter((task)=> task.assignedPersons?.includes(loggedInpersonId ));
+      }
+      
+      setTasks(fetchedTasks);
+      const fetchedTimeEntries = await Promise.all(fetchedTasks.map(async (task) => {
+        try {
+          if (project.id ){           
+            const totalTimeEntries = await timeEntriesApi.listProjectTimeEntries({ projectId: project.id, taskId: task.id });
+            let totalMinutes = 0;
+            totalTimeEntries.forEach((timeEntry: TimeEntries)=> {
+              if (loggedInpersonId && timeEntry.person===loggedInpersonId) {
+                totalMinutes+=(timeEntry.timeRegistered || 0)
+              }
+              if (!loggedInpersonId) {
+                totalMinutes += timeEntry.timeRegistered || 0;
+              }
+            })
+            return totalMinutes;
+          }
+          return 0;
+        } catch (error) {
+          console.error(`Error fetching time entries for task ${task.id}: ${error}`);
+          return 0;
+        }
+      }));
+      setTimeEntries(fetchedTimeEntries);
+    } catch (error) {
+      console.error(`Error fetching time entries: ${error}`);
+    }
+  };
+
+  /**
+   * Retrieve total time entries for a task
+   */
+  const getTotalTimeEntries = (task: Tasks) => {
+    if (timeEntries.length!==0) {
+      return timeEntries[tasks.indexOf(task)];
+    }
+    return 0;
+  }
+
+  /**
+   * Lowercase all letters in a string except the first letter except first one
```

Comment:
Missing @param docs

Reply:
Will be removed, there is no need to have this function and docs